### PR TITLE
move the api classes

### PR DESCRIPTION
**Pull request by kinggoesgaming**
move the API classes to src/api/java.

The structure will not break as the final result is the same module to the dependents.

### CODE REVIEW

**Comment by Flibio** [comment]
Does this change the permission for notifications? Or is there now a Sponge permission prefix I haven't heard about?

**Comment by kinggoesgaming** [comment]
don't understand your question...

File: src/main/java/me/flibio/updatifier/PluginInfo.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
     public static final String ID = "@project.id@";
     public static final String NAME = "@project.name@";
     public static final String VERSION = "@project.version@";
-    public static final String PERM_NOTIFY = "updatifier.notify";
+    public static final String PERM_NOTIFY = "me.flibio.updatifier.notify";
```
Comment (Flibio) [inline src/main/java/me/flibio/updatifier/PluginInfo.java]: Right here.

Comment (kinggoesgaming) [inline src/main/java/me/flibio/updatifier/PluginInfo.java]: damn it... IntelliJ refactored my strings wrong :/

**Comment by kinggoesgaming** [comment]
@Flibio should be good now

**Comment by Flibio** [comment]
Approved. Looks ready to roll.

**Comment by Flibio** [comment]
You need to refactor against master.

**Pull request merged**